docs/ui/Do.tsx

```diff
@@ -0,0 +1,30 @@
+import { Text } from '@marigold/components';
+
+import { Image } from '@/ui/Image';
+
+interface Props {
+  img: string;
+  description: string;
+}
+
+export const Do = ({ description, img }: Props) => {
+  return (
+    <div className="border-border flex flex-col border">
```

Comment:
use a card as a rapper? this would fit niceley with all the other stuff we have :)

Reply:
Now there is discussion about the Card to use it or not 
https://marigold-docs-git-do-card-marigold.vercel.app/components/content/card?vercelThreadId=kuiD6
@sebald